tests/html5skin-test.js

```diff
@@ -1721,6 +1721,23 @@ describe('Controller', function() {
 
   });
 
+  describe('AirPlay button', () => {
+    window.WebKitPlaybackTargetAvailabilityEvent = true;
```

Comment:
will it be set to true in all other tests?

Reply:
I think yes

Reply:
so you should set back to `false`, to avoid unexpected tests failure

Reply:
ok

Reply:
please do it in `beforeAll` and `afterAll` inside current `describe`

Reply:
Thanks, done.